fluent-syntax/src/parser.js

```diff
@@ -692,10 +692,14 @@ export default class FluentParser {
     ps.expectChar('"');
 
     let ch;
-    while ((ch = ps.takeChar(x => x !== '"'))) {
+    while ((ch = ps.takeChar(x => x !== '"' && x !== '\n'))) {
       val += ch;
     }
 
+    if (ps.currentIs('\n')) {
+      throw new ParseError('E0020');
```

Comment:
I submitted https://github.com/projectfluent/fluent/pull/83 to reflect this change in the spec.